java/ql/lib/change-notes/2024-09-20-kotlin-2.1.0.md

```diff
@@ -0,0 +1,4 @@
+---
+category: feature
+---
+* Kotlin versions up to 2.1.0\ *x* are now supported.
```

Comment:
I don't understand this `2.1.0\ *x*` version. Shouldn't we instead say we support versions up to `2.1.*`?

Reply:
Kotlin doesn't guarantee that interfaces won't change between e.g. 2.1.0 and 2.1.10.

This text makes more sense for `2.1.1x`, where the final component is 2 digits. The intention here is to include e.g. `2.1.5`, but that is unfortunately not written `2.1.05`. I'm not sure we can concisely write something better, though.